habitat-baselines/habitat_baselines/rl/ppo/ppo_trainer.py

```diff
@@ -1038,13 +907,13 @@ def _eval_checkpoint(
             # NB: Move actions to CPU.  If CUDA tensors are
             # sent in to env.step(), that will create CUDA contexts
             # in the subprocesses.
-            if is_continuous_action_space(self.env_action_space):
+            if is_continuous_action_space(self._env_spec.action_space):
```

Comment:
nit, make is_continuous_action_space a method on EnvironmentSpec so that the `is_continuous_action_space` is not isolated.